src/main/resources/uischema.json

```diff
@@ -139,208 +141,218 @@
             }
           },
           {
             "type": "attribute",
             "value": {
-              "name": "offset.storage.replication.factor",
-              "displayName": "Offset Storage Replication Factor",
+              "name": "connector.name",
+              "displayName": "Connector Name",
               "inputType": "string",
               "required": "false",
-              "helpTip": "Replication factor used when creating the offset storage topic. Required when offset.storage is set to the <…>.KafkaOffsetBackingStore."
+              "helpTip": "Unique name for the connector."
             }
           },
           {
             "type": "attribute",
             "value": {
-              "name": "database.hostname",
-              "displayName": "Database Hostname",
+              "name": "topic.prefix",
+              "displayName": "Topic Prefix",
               "inputType": "string",
-              "required": "true",
-              "helpTip": "IP address or host name of the database server."
+              "required": "false",
+              "helpTip": "Topic prefix that provides a namespace for the database server that you want Debezium to capture. The prefix should be unique across all other connectors, since it is used as the prefix for all Kafka topic names that receive records from this connector. Only alphanumeric characters, hyphens, dots, and underscores must be used in the database server logical name."
             }
           },
           {
             "type": "attribute",
             "value": {
-              "name": "database.port",
-              "displayName": "Database Port",
+              "name": "schema.history.internal",
+              "displayName": "Schema History Internal",
               "inputType": "string",
-              "required": "true",
-              "helpTip": "Port number (Integer) of the database server."
+              "required": "false",
+              "helpTip": "The name of the Java class that is responsible for persistence of the database schema history. It must implement <…>.SchemaHistory interface."
             }
           },
           {
             "type": "attribute",
             "value": {
-              "name": "database.user",
-              "displayName": "Database User",
+              "name": "schema.history.internal.file.filename",
+              "displayName": "Schema History Internal File Filename",
               "inputType": "string",
-              "required": "true",
-              "helpTip": "Name of the database user to use when connecting to the database server."
+              "required": "false",
+              "helpTip": "This value is required only if io.debezium.storage.file.history.FileSchemaHistory was provided for the schema.history.internal value. You need to specify the path to a file where the database schema history is stored. By default, the file will be stored in the <Product home>/cdc/schemaHistory directory."
```

Comment:
Shall we use conditional rendering here?

Reply:
Sure